docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -72,12 +72,12 @@ space. While the exact amount resource usage of each additional field depends
 on the type of field, an additional 1024B in memory use  for each additional
 mapped field on data nodes is a good approximation in most cases.
 As a result, the more mapped fields you have per index, the fewer
-indices and shards will each individual data node be able to hold,
+indices and shards each individual data node will be able to hold,
 all other things being equal. In addition to the per field overhead you should
 at least have an additional 512MB of heap available per data node.
 For example, holding 1000 indices that each contain 4000 fields will require
```

Comment:
```suggestion
For example, holding 1000 indices that each contains 4000 fields will require
```
Should this be contains following each being a singular?

Reply:
Don't know how to explain this one, but `contain` is right to my ear. I might say "holding 1000 indices each containing 4000 fields" instead.

Reply:
I looked it up: in this construction `each` is an adverb modifying the verb `contain` rather than being its subject. The indices are the subject, so the verb is plural.